### PR TITLE
Ingested AMTA 2008

### PR DESCRIPTION
**Pull request by davidstap**
For original see https://aclanthology.org/www.mt-archive.info/05/AMTA-2008-TOC.htm

I have listed the two papers at the bodem ("Additional papers not in proceedings") separately. Not sure if they should be included at all?

### CODE REVIEW

File: data/xml/2008.amta.xml [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+    <paper id="1">
+      <title><fixed-case>S</fixed-case>panish-to-<fixed-case>B</fixed-case>asque <fixed-case>M</fixed-case>ulti<fixed-case>E</fixed-case>ngine Machine Translation for a Restricted Domain</title>
+      <author><first>Iñaki</first><last>Alegria</last></author>
+      <author><first>Arantza</first><last>Casillas</last></author>
+      <author><first>Arantza</first><last>Diaz de Ilarraza</last></author>
+      <author><first>Jon</first><last>Igartua</last></author>
+      <author><first>Gorka</first><last>Labaka</last></author>
+      <author><first>Mikel</first><last>Lersundi</last></author>
+      <author><first>Aingeru</first><last>Mayor</last></author>
+      <author><first>Alegria</first><last>Kepa SarasolaIñaki</last></author>
```
Comment (danielgildea) [inline data/xml/2008.amta.xml:51]: author list duplicated

File: data/xml/2008.amta.xml [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+      <author><first>Arantza</first><last>Casillas</last></author>
+      <author><first>Arantza</first><last>Diaz de Ilarraza</last></author>
+      <author><first>Jon</first><last>Igartua</last></author>
+      <author><first>Gorka</first><last>Labaka</last></author>
+      <author><first>Mikel</first><last>Lersundi</last></author>
+      <author><first>Aingeru</first><last>Mayor</last></author>
+      <author><first>Kepa</first><last>Sarasola</last></author>
+      <url hash="0cffa517">2008.amta-papers.1</url>
+      <pages>37-45</pages>
+      <abstract>  We present our initial strategy for Spanish-to-Basque MultiEngine Machine Translation, a language pair with very different structure and word order and with no huge parallel corpus available. This hybrid proposal is based on the combination of three different MT paradigms: Example-Based MT, Statistical MT and Rule- Based MT. We have evaluated the system, reporting automatic evaluation metrics for a corpus in a test domain. The first results obtained are encouraging.</abstract>
```
Comment (mjpost) [inline data/xml/2008.amta.xml:61]: Weird space here in the abstract.

File: data/xml/2008.amta.xml [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+    <paper id="3">
+      <title>Machine Translation 2008: Science meets Solution</title>
+      <author><first>Mark</first><last>Tapling</last></author>
+      <url hash="b554a117">2008.amta-keynotes.3</url>
+      <pages>23-36</pages>
+    </paper>
+  </volume>
+  <volume id="papers" ingest-date="2021-05-19">
+    <meta>
+      <booktitle>Proceedings of the 6th Conference of the Association for Machine Translation in the Americas: Technical Papers</booktitle>
```
Comment (mjpost) [inline data/xml/2008.amta.xml:33]: Eighth Conference

Comment (mjpost) [reply data/xml/2008.amta.xml:33]: Also, where did you get "Technical Papers" from? From the website, I might put "Research Papers"

File: data/xml/2008.amta.xml [hunk 4]
```diff
@@ -0,0 +1,543 @@
...
+    <paper id="21">
+      <title>Two-Stage Translation: A Combined Linguistic and Statistical Machine Translation Framework</title>
+      <author><first>Yushi</first><last>Xu</last></author>
+      <author><first>Stephanie</first><last>Seneff</last></author>
+      <url hash="24f8a916">2008.amta-papers.21</url>
+      <pages>222-231</pages>
+      <abstract>We propose a two-stage system for spoken language machine translation. In the first stage, the source sentence is parsed and paraphrased into an intermediate language which retains the words in the source language but follows the word order of the target language as much as feasible. This stage is mostly linguistic. In the second stage, a statistical MT is performed to translate the intermediate language into the target language. For the task of English-to-Mandarin translation, we achieved a 2.5 increase in BLEU score and a 45% decrease in GIZA-Alignment Crossover, on IWSLT-06 data. In a human evaluation of the sentences that differed, the two-stage system was preferred three times as often as the baseline.</abstract>
+    </paper>
+  </volume>
+  <volume id="studentworkshop" ingest-date="2021-05-19">
```
Comment (mjpost) [inline data/xml/2008.amta.xml:238]: We typically use "srw" for student research workshops.

File: data/xml/2008.amta.xml [hunk 5]
```diff
@@ -0,0 +1,543 @@
...
+    </paper>
+  </volume>
+  <volume id="govandcom" ingest-date="2021-05-19">
+    <meta>
+      <booktitle>Proceedings of the 6th Conference of the Association for Machine Translation in the Americas: Government and Commercial Uses of MT</booktitle>
```
Comment (mjpost) [inline data/xml/2008.amta.xml:296]: Eight Conference

File: data/xml/2008.amta.xml [hunk 6]
```diff
@@ -0,0 +1,543 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2008.amta">
+  <volume id="keynotes" ingest-date="2021-05-19">
+    <meta>
+      <booktitle>Proceedings of the 6th Conference of the Association for Machine Translation in the Americas: Keynote Presentations</booktitle>
```
Comment (mjpost) [inline data/xml/2008.amta.xml:5]: Eighth Conference

File: data/xml/2008.amta.xml [hunk 7]
```diff
@@ -0,0 +1,543 @@
...
+      <author><first>Stephanie</first><last>Seneff</last></author>
+      <url hash="24f8a916">2008.amta-papers.21</url>
+      <pages>222-231</pages>
+      <abstract>We propose a two-stage system for spoken language machine translation. In the first stage, the source sentence is parsed and paraphrased into an intermediate language which retains the words in the source language but follows the word order of the target language as much as feasible. This stage is mostly linguistic. In the second stage, a statistical MT is performed to translate the intermediate language into the target language. For the task of English-to-Mandarin translation, we achieved a 2.5 increase in BLEU score and a 45% decrease in GIZA-Alignment Crossover, on IWSLT-06 data. In a human evaluation of the sentences that differed, the two-stage system was preferred three times as often as the baseline.</abstract>
+    </paper>
+  </volume>
+  <volume id="studentworkshop" ingest-date="2021-05-19">
+    <meta>
+      <booktitle>Proceedings of the 6th Conference of the Association for Machine Translation in the Americas: Student Research Workshop</booktitle>
```
Comment (mjpost) [inline data/xml/2008.amta.xml:240]: Eighth Conference

File: data/xml/2008.amta.xml [hunk 8]
```diff
@@ -0,0 +1,543 @@
...
+      <author><first>Chengqing</first><last>Zong</last></author>
+      <author><first>Heyan</first><last>Huang</last></author>
+      <author><first>Shuming</first><last>Shi</last></author>
+      <url hash="17ccaae6">2008.amta-govandcom.26</url>
+      <pages>470-479</pages>
+    </paper>
+  </volume>
+  <volume id="additional" ingest-date="2021-05-19">
+    <meta>
+      <booktitle>Additional Papers not in Proceedings</booktitle>
```
Comment (mjpost) [inline data/xml/2008.amta.xml:524]: Let's call this "Proceedings of the Eighth Conference of the Association for Machine Translation in the Americas: Additional Papers"

File: data/xml/2008.amta.xml [hunk 9]
```diff
@@ -0,0 +1,543 @@
...
+      <pages>163-171</pages>
+      <abstract>In this paper, we propose a probabilistic phrase alignment model based on dependency trees. This model is linguistically-motivated, using syntactic information during alignment process. The main advantage of this model is that the linguistic difference between source and target languages is successfully absorbed. It is composed of two models: Model1 is using content word translation probability and function word translation probability; Model2 uses dependency relation probability which is defined for a pair of positional relations on dependency trees. Relation probability acts as tree-based phrase reordering model. Since this model is directed, we combine two alignment results from bi-directional training by symmetrization heuristics to get definitive alignment. We conduct experiments on a Japanese-English corpus, and achieve reasonably high quality of alignment compared with word-based alignment model.</abstract>
+    </paper>
+    <paper id="16">
+      <title>Parsers as language models for statistical machine translation</title>
+      <author><first>Matt</first><last>Post</last></author>
```
Comment (mjpost) [inline data/xml/2008.amta.xml:186]: Aw shucks, my first paper

File: data/xml/2008.amta.xml (hunk 3 above)
Comment (davidstap) [reply data/xml/2008.amta.xml:33]: I chose "Technical Papers" to be consistent with previous volumes. I'll switch to "Research Papers".

File: data/xml/2008.amta.xml (hunk 1 above)
Comment (davidstap) [reply data/xml/2008.amta.xml]: thanks for spotting @danielgildea, not sure what happened here.

File: data/xml/2008.amta.xml (hunk 9 above)
Comment (davidstap) [reply data/xml/2008.amta.xml:179]: haha such a long time ago! ;-)

**Comment by davidstap** [comment]
this seems ready to merge, or am I missing something? Can you approve @mjpost?

**Review by mjpost — APPROVED** [review]
LGTM

**Pull request merged**